.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
```

Comment:
Is the space after the bracket intentional?

Reply:
It was, that way when the cursor is in the title the user doesn’t have to add the space before typing the description